brightlayer-ui/react-workflows
# Refactor shared styles to avoid injection order issues

Fixes an issue where the styles are broken when changing the import syntax to reduce bundle size


#### Changes proposed in this Pull Request:
- Simplify some style function imports
- use createStyles in each component separately so that the style tag appears at the bottom of the <head> and isn't overridden by default styles


#### To Test:
- Use these changes in the demo project
- Generate a production build
- Examine the final bundle size and see that it is under 3MB
- Examine the demo project and ensure that the styles look correct throughout all of the screens
